package.json

```diff
@@ -63,8 +63,8 @@
     "is-github-url": "^1.2.2",
     "js-cookie": "^2.2.1",
```

Comment:
Please check all features in application where event pooling is used -> 
React 17.0 removed event pooling and check there are no breaking changes to current implementations in app.

If all working correct give me info and we will merge this

Reply:
Zapoznałem się z zagadnieniem event poolingu, ale nie jestem pewny czy w pełni go zrozumiałem. Sprawdziłem działanie aplikacji w wielu miejscach i nie było problemów.